unity-renderer/Assets/Rendering/Shaders/Toon/Master_ToonShader.shadergraph

```diff
@@ -200,7 +200,10 @@
             "m_Id": "6b35ac670faf4e3886f2b8ca78e83469"
```

Comment:
The shadergraph changes are really needed? Otherwise I'm going to have to resolve conflicts again on #785

Reply:
Yes, now the dither is applied as alpha clip threshold :(.

Reply:
So i'd have to hook the node output to the alpha clip threshold instead of alpha now? That's the only change? (if not, i'd need a screenshot).

Reply:
I will do it in your branch :)

Reply:
Outdated